queries.sql

```diff
@@ -1,5 +1,88 @@
-/* Populate database with sample data. */
+/*Queries that provide answers to the questions from all projects.*/
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
\ No newline at end of file
+SELECT * from animals WHERE name = 'Pikachu';
+
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = true;
+
+SELECT * FROM animals WHERE name <> 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+-- Transactions begin
+BEGIN;
+UPDATE animals
+SET species='unspecified';
+SELECT * FROM animals
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species='pokemon'
+WHERE species='';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT my_savepoint;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO my_savepoint
+SELECT * FROM animals;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+SELECT * FROM animals;
+
+SELECT COUNT(name) FROM animals;
+SELECT COUNT(name) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT SUM(escape_attempts) FROM animals
+WHERE neutered = True;
+
+SELECT SUM(escape_attempts) FROM animals
+WHERE neutered = False;
```

Comment:
- [ Required ] I think it would be better if you used a single query to answer the question _**Who escapes the most, neutered or not neutered animals?**_. Using two queries makes it difficult to compare whether neutered or not neutered animals escape the most because the results appear in two places and not one.

## Example code 
```sql
  SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```